shell/browser/api/electron_api_web_contents.cc

```diff
@@ -3134,6 +3127,17 @@ void WebContents::StartDrag(const gin_helper::Dictionary& item,
   }
 }
 
+void WebContents::OnCapturePageDone(gin_helper::Promise<gfx::Image> promise,
+                                    const SkBitmap& bitmap) {
+  // Hack to enable transparency in captured image
+  promise.Resolve(gfx::Image::CreateFrom1xBitmap(bitmap));
+  if (!capture_handles_.empty()) {
+    base::ScopedClosureRunner handle = std::move(capture_handles_.front());
+    capture_handles_.pop();
+    handle.RunAndReset();
+  }
```

Comment:
An unresolved (un-reset) ScopedClosureRunner automatically gets run in its dtor, so removing it from the queue should be enough w/o having to call `RunAndReset()` explicitly, so

```suggestion
  if (!capture_handles_.empty()) {
    capture_handles_.pop();
  }
```

Is all that's necessary. However the current PR's version is arguably more readable?